src/main/java/equipments/technology/AbstractTechnologyEquipment.java

```diff
@@ -0,0 +1,22 @@
+package equipments.technology;
+
+import equipments.AbstractEquipment;
+
+public abstract class AbstractTechnologyEquipment extends AbstractEquipment {
+    public String IP;
+    public AbstractTechnologyEquipment(String IP){
+        this.IP = IP;
```

Comment:
Code Duplication- why not call `setIP`? if somewhere down the line you add some sort of validation to make sure it's a valid IP before setting it, make sure you can't bypass it